pkg/setting/date_formats.go

```diff
@@ -0,0 +1,28 @@
+package setting
+
+type DateFormats struct {
+	FullDate         string              `json:"fullDate"`
+	UseBrowserLocale bool                `json:"useBrowserLocale"`
+	Interval         DateFormatIntervals `json:"interval"`
+}
+
+type DateFormatIntervals struct {
+	Second string `json:"second"`
+	Minute string `json:"minute"`
+	Hour   string `json:"hour"`
+	Day    string `json:"day"`
+	Month  string `json:"month"`
+	Year   string `json:"year"`
+}
+
+func (cfg *Cfg) readDateFormats() {
+	preferences := cfg.Raw.Section("date_formats")
```

Comment:
Nit. Could rename preferences to something "date formats section".

Reply:
fixed